### DISCUSSION TITLE
What are the alternative payment gateways that can be integrated with GitHub Sponsors for countries where Stripe is not available?

### DISCUSSION THREAD

**Discussion by nadeeshaofficial1** [community/community#132093] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Help me !

**Comment by Yewzir** [comment, marked as answer]
@nadeeshaofficial1 

PayPal used to be one of them, but is no longer supported by GitHub, unfortunately. As alternatives, you can either link a **debit or credit card**, or link a **Patreon** account.

For additional information on both alternatives, the following sections from the official GitHub documentation on GitHub Sponsors will help you set things up in no time.

[Sponsoring an open source contributor through GitHub](https://docs.github.com/en/sponsors/sponsoring-open-source-contributors/sponsoring-an-open-source-contributor-through-github)
[Sponsoring an open source contributor through Patreon](https://docs.github.com/en/sponsors/sponsoring-open-source-contributors/sponsoring-an-open-source-contributor-through-patreon)

🎉 

Kind regards
Yewzir

**Reply by nadeeshaofficial1** [reply]
Thank u brother
